### ISSUE TITLE
Continuation indent lint

### ISSUE THREAD

**Issue by fracz** [htmllint/htmllint#235]
I would like this to lint correctly:

```html
<div>
    <img src="..."
        alt="">
</div>
```

And this not:

```html
<div>
    <img src="..."
               alt="">
</div>
```

How can I achieve this?

My current config enforces one attribute per line correctly, but it ignores their indentation.

```json
{
  "attr-new-line": 1,
  "indent-style": "spaces",
  "indent-width-cont": true,
}
```

**Comment by fracz** [comment]
Just figured it out: need to set exactly the opposite `"indent-with-cont": false` and it works as expected. https://github.com/htmllint/htmllint/issues/117
